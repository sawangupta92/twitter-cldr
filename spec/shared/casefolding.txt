Dreiländereck
aus Wikipedia, der freien Enzyklopädie
Wechseln zu: Navigation, Suche
Dreiländereck D/F/CH am Rheinhafen von Basel
Vaalserberg: Dreiländereck (D/NL/B)
Sevening: Europadenkmal (D/B/L)
Denkmal für das „Schengener Abkommen“ in Schengen, Luxemburg (D/L/F)
Zittau: Dreiländereck (D/CZ/PL)
Treriksröset: Dreiländereck FIN/N/S
Berg Ofen: Grenzstein Dreiländereck I/SLO/A
Monte Roraima: Dreiländereck Brasilien, Guyana, Venezuela
Markstein auf der argentinischen Seite des Dreiländerecks RA/BR/PY

Ein Dreiländereck – auch Länderdreieck genannt – ist der geografische Punkt, wo drei gleichrangige Grenzen und auch drei gleich geartete Territorien (Länder, Staaten, Bundesländer) aufeinander treffen. Bekannt werden solche Dreiländerecke meist, wenn sie schon längere Zeit bestehen oder aber, wenn drei Kulturen aufeinander treffen.

Meist ist diese Stelle durch die jeweiligen Staatsflaggen oder Monumente markiert.

Von den zurzeit 195 Staaten auf der Erde haben 134 einen solchen Grenzpunkt. Insgesamt existieren 157 Dreiländerecke auf der Erde. Die Häufigkeit des Vorkommens von Dreiländerecken ist abhängig von der Anzahl der Nachbarn, aber auch von der Anzahl der Seegrenzen. So haben Inselnationen wie Japan keines, wie auch Staaten wie Portugal oder Dänemark, die nur ein Nachbarland haben bzw. vollständig von einem Staat umgeben sind (wie z. B. San Marino). Auch sind zwei Nachbarländer kein hinreichendes Kriterium, wie es z. B. bzgl. der USA in Nordamerika ist. Auch die Fläche eines Staats hat nur bedingt mit der Anzahl an Dreiländerecken zu tun: Der Staat mit den meisten ist China mit 15, während Russland, der flächenmäßig größte Staat der Welt, nur acht hat.

Österreich hat an seinen Grenzen neun Dreiländerecke (und ist damit der Staat mit den meisten Dreiländerecken innerhalb Europas), Deutschland sieben, die Schweiz sechs und Liechtenstein weist zwei Dreiländerecke auf.
Inhaltsverzeichnis
[Verbergen]

    * 1 Dreiländerecke weltweit
          o 1.1 Europa
          o 1.2 Mittel- und Südamerika
          o 1.3 Nordamerika
          o 1.4 Afrika
          o 1.5 Asien
    * 2 Ehemalige Dreiländerecke
          o 2.1 Europa
    * 3 Siehe auch
    * 4 Weblinks

Dreiländerecke weltweit [Bearbeiten]

Alphabetische Liste von allen 46 internationalen Dreiländerecken in Europa:

Europa [Bearbeiten]

    * Albanien – Griechenland – Mazedonien, am Prespa-See
    * Albanien – Mazedonien – Kosovo, südlich von Prizren in Kosovo
    * Albanien – Montenegro – Kosovo, nördlich von Bajram Curri in Albanien
    * Andorra – Frankreich – Spanien (Osteck), in den Pyrenäen
    * Andorra – Frankreich – Spanien (Westeck), in den Pyrenäen
    * Belgien – Deutschland – Luxemburg, am Europadenkmal an der Our bei Sevenig
    * Belgien – Deutschland – Niederlande, auf dem Vaalserberg westlich von Aachen (war zeitweilig auch wegen Neutral-Moresnet ein Vierländereck)
    * Belgien – Frankreich – Luxemburg, bei Longwy, Pétange, Aubange
    * Bosnien-Herzegowina – Kroatien – Montenegro, nordöstlich von Gruda in Kroatien
    * Bosnien-Herzegowina – Kroatien – Serbien, westlich von Omjelovo in Bosnien-Herzegowina
    * Bosnien-Herzegowina – Montenegro – Serbien, westlich von Priboj in Serbien
    * Bulgarien – Griechenland – Mazedonien, westlich von Petritsch in Bulgarien
    * Bulgarien – Griechenland – Türkei, bei der griechischen Gemeinde Trigono, westlich des türkischen Edirne
    * Bulgarien – Mazedonien – Serbien, westlich von Kjustendil in Bulgarien
    * Bulgarien – Rumänien – Serbien, östlich von Negotin in Serbien
    * Deutschland – Frankreich – Luxemburg, bei Perl, Apach, Schengen in der Mosel
    * Deutschland – Frankreich – Schweiz, bei Basel, Hüningen und Weil am Rhein im Rhein
    * Deutschland – Polen – Tschechien, südöstlich von Zittau
    * Deutschland – Schweiz – Österreich, im Bodensee
    * Deutschland – Tschechien – Österreich, am Plöckenstein (1320 m), Böhmerwald
    * Estland – Lettland – Russland, westlich von Stuborova in Lettland
    * Finnland – Norwegen – Russland, nordöstlich von Ivalo in Finnland
    * Finnland – Norwegen – Schweden, schwedisch Treriksröset, norwegisch Treriksrøysa, finnisch Kolmen valtakunnan rajapyykki, westlich von Kilpisjärvi in Finnland
    * Frankreich – Italien – Schweiz, auf der Bergspitze des Mont Dolent (3820 m) nahe dem Mont Blanc
    * Italien – Schweiz – Österreich, nähe Reschenpass an der „Dreiländerecke“ (2.179 m)
    * Italien – Slowenien – Österreich, bei Arnoldstein auf dem Gipfel des Ofen (slow. Peč) alias Dreiländereck
    * Kroatien – Serbien – Ungarn, nordöstlich von Gruda in Kroatien
    * Kroatien – Slowenien – Ungarn, südöstlich von Letenye in Ungarn
    * Lettland – Litauen – Weißrussland, nordöstlich von Visaginas in Litauen
    * Lettland – Russland – Weißrussland, südöstlich von Suskova in Lettland
    * Liechtenstein – Schweiz – Österreich (Nordeck), im Rhein bei Sennwald
    * Liechtenstein – Schweiz – Österreich (Südeck), im Rätikon am Naafkopf (2571 m)
    * Litauen – Polen – Russland (Kaliningrad), westlich von Wizajny in Polen
    * Litauen – Polen – Weißrussland, westlich von Druskininkai in Litauen
    * Moldawien – Rumänien – Ukraine (Nordeck), nordöstlich von Dărăbani in Rumänien
    * Moldawien – Rumänien – Ukraine (Südeck), östlich von Galaţi in Rumänien
    * Polen – Slowakei – Tschechien, bei Hrčava in Tschechien und Jaworzynka in Polen
    * Polen – Slowakei – Ukraine, nordöstlich von Nova Sedlica in Slowakei
    * Polen – Ukraine – Weißrussland, südöstlich von Wlodawa in Polen
    * Rumänien – Serbien – Ungarn, südlich von Kübekhaza in Ungarn
    * Rumänien – Ukraine – Ungarn, nordöstlich von Bercu in Rumänien
    * Russland – Ukraine – Weißrussland, südöstlich von Homel in Weißrussland
    * Slowakei – Ukraine – Ungarn, bei Zahony in Ungarn
    * Slowakei – Ungarn – Österreich, bei Deutsch Jahrndorf in Österreich, Rajka in Ungarn
    * Slowakei – Tschechien – Österreich, bei Hohenau an der March in Österreich
    * Slowenien – Österreich – Ungarn, östlich von Tauka in Österreich

Die nachfolgende Auflistung zeigt eine kleine Auswahl existierender Dreiländerecke in anderen Kontinenten:

Mittel- und Südamerika [Bearbeiten]

In Mittelamerika existieren zwei Dreiländerecke und in Südamerika 13:

    * Argentinien – Bolivien – Chile
    * Argentinien – Bolivien – Paraguay
    * Argentinien – Brasilien – Paraguay: Hito Tres Fronteras, bekannt durch die Iguaçu-Wasserfälle
    * Argentinien – Brasilien – Uruguay
    * Mittelamerika: Belize – Guatemala – Mexiko
    * Bolivien – Brasilien – Paraguay
    * Bolivien – Brasilien – Peru
    * Bolivien – Chile – Peru
    * Brasilien – Kolumbien – Peru: Tres Fronteras, im Amazonas-Regenwald mit einer Dreistadt Tabatinga, Leticia, Santa Rosa mit mehr als 100.000 Einwohnern
    * Brasilien – Kolumbien – Venezuela
    * Brasilien – Französisch-Guayana – Surinam
    * Brasilien – Guyana – Surinam
    * Brasilien – Guyana – Venezuela: Auf dem Berg Monte Roraima
    * Kolumbien – Ecuador – Peru
    * Mittelamerika: Guatemala – Honduras – El Salvador

Nordamerika [Bearbeiten]

Da mit Kanada und den Vereinigten Staaten nur zwei Staaten in Nordamerika neben einigen französischen Inseln an der kanadischen Ostküste existieren, kann es dort keine Dreiländerecke geben.

Die Bundesstaaten der Vereinigten Staaten bilden 38 Tri-State Areas (Dreistaatengebiete).

Afrika [Bearbeiten]

In Afrika existieren 55 Dreiländerecke:

    * Südafrika – Botswana – Namibia
    * Südafrika – Botswana – Simbabwe
    * Südafrika – Mosambik – Simbabwe (siehe Crook’s Corner)
    * Südafrika – Mosambik – Swasiland (Nord)
    * Südafrika – Mosambik – Swasiland (Süd)
    * Algerien – Libyen – Niger
    * Algerien – Libyen – Tunesien
    * Algerien – Mali – Mauretanien
    * Algerien – Mali – Niger
    * Algerien – Marokko – West-Sahara
    * Algerien – Mauretanien – West-Sahara
    * Angola – Namibia – Sambia
    * Angola – Republik Kongo – Demokratische Republik Kongo: Cabinda
    * Angola – Demokratische Republik Kongo – Sambia
    * Benin – Burkina Faso – Niger
    * Benin – Burkina Faso – Togo
    * Benin – Niger – Nigeria
    * Botswana – Sambia – Simbabwe
    * Burkina Faso – Elfenbeinküste – Ghana
    * Burkina Faso – Elfenbeinküste – Mali
    * Burkina Faso – Ghana – Togo
    * Burkina Faso – Mali – Niger
    * Burundi – Demokratische Republik Kongo – Ruanda
    * Burundi – Ruanda – Tansania
    * Kamerun – Gabun – Äquatorial-Guinea
    * Kamerun – Gabun – Republik Kongo
    * Kamerun – Nigeria – Tschad
    * Kamerun – Zentralafrikanische Republik – Republik Kongo
    * Kamerun – Zentralafrikanische Republik – Tschad
    * Elfenbeinküste – Guinea – Liberia
    * Elfenbeinküste – Guinea – Mali
    * Dschibuti – Eritrea – Äthiopien
    * Dschibuti – Äthiopien – Somalia
    * Ägypten – Libyen – Sudan
    * Eritrea – Äthiopien – Sudan
    * Äthiopien – Kenia – Somalia
    * Äthiopien – Kenia – Sudan
    * Guinea – Guinea-Bissau – Senegal
    * Guinea – Liberia – Sierra Leone
    * Kenia – Uganda – Sudan
    * Kenia – Uganda – Tansania
    * Libyen – Niger – Tschad
    * Libyen – Sudan – Tschad
    * Malawi – Mosambik – Tansania
    * Malawi – Mosambik – Sambia
    * Mosambik – Tansania – Sambia
    * Mosambik – Sambia – Simbabwe: Bei Zumbo
    * Mali – Mauretanien – Senegal
    * Niger – Nigeria – Tschad
    * Uganda – Demokratische Republik Kongo – Ruanda
    * Uganda – Demokratische Republik Kongo – Sudan
    * Uganda – Ruanda – Tansania
    * Zentralafrikanische Republik – Republik Kongo – Demokratische Republik Kongo
    * Zentralafrikanische Republik – Demokratische Republik Kongo – Sudan
    * Demokratische Republik Kongo – Tansania – Sambia

Asien [Bearbeiten]

In Asien existieren 42 Dreiländerecke:

    * Afghanistan – Iran – Pakistan
    * Afghanistan – Iran – Turkmenistan
    * Afghanistan – VR China – Pakistan
    * Afghanistan – VR China – Tadschikistan
    * Afghanistan – Usbekistan – Tadschikistan
    * Afghanistan – Usbekistan – Turkmenistan
    * Saudi-Arabien – Vereinigte Arabische Emirate – Oman: Umm az Zamul
    * Saudi-Arabien – Irak – Jordanien
    * Saudi-Arabien – Irak – Kuwait
    * Saudi-Arabien – Oman – Jemen
    * Armenien – Aserbaidschan – Georgien

Mekong und Goldenes Dreieck

    * Armenien – Aserbaidschan – Iran
    * Armenien – Aserbaidschan – Iran
    * Armenien – Aserbaidschan – Türkei
    * Gaza-Streifen – Ägypten – Israel
    * Bangladesch – Burma – Indien
    * Bhutan – VR China – Indien (Osten)
    * Bhutan – VR China – Indien (Westen)
    * Burma – VR China – Indien
    * Burma – VR China – Laos
    * Burma – Laos – Thailand: Goldenes Dreieck
    * Kambodscha – Laos – Thailand
    * Kambodscha – Laos – Vietnam
    * VR China – Indien – Nepal (Osten)
    * VR China – Indien – Nepal (Westen)
    * VR China – Indien – Pakistan
    * VR China – Kasachstan – Kirgisien
    * VR China – Kasachstan – Russland
    * VR China – Laos – Vietnam
    * VR China – Mongolei – Russland (Osten)
    * VR China – Mongolei – Russland (Westen)
    * VR China – Nord-Korea – Russland in der Region Primorje
    * Palästina – Israel – Jordanien (Nord)
    * Palästina – Israel – Jordanien (Süden)
    * Irak – Iran – Türkei
    * Irak – Jordanien – Syrien
    * Irak – Syrien – Türkei
    * Israel – Jordanien – Syrien
    * Israel – Libanon – Syrien: Mount Hermon
    * Kasachstan – Kirgisien – Usbekistan
    * Kasachstan – Usbekistan – Turkmenistan
    * Kirgisien – Usbekistan – Tadschikistan

Die Dreiländerecke VR China – Kasachstan – Russland sowie VR China – Mongolei – Russland liegen im Altai nur 50 Kilometer voneinander entfernt.

Ehemalige Dreiländerecke [Bearbeiten]

Infolge von Grenzverschiebungen kommt es vor, dass Dreiländerecke zu normalen Staatsgrenzen werden, aber dennoch ein markanter Punkt bleiben:

Europa [Bearbeiten]

    * Italien – Schweiz – Österreich; bis zur Abtretung Südtirols von Österreich an Italien, nähe Stilfserjoch auf der Dreisprachenspitze. Es ist heute ein „Dreispracheneck“ geblieben mit Italienisch (Lombardei), Deutsch (Südtirol) und Rätoromanisch (Schweiz).
    * BR Deutschland – DDR – Tschechoslowakei; bis zur Wiedervereinigung an der Stelle, wo Bayern, Sachsen und Böhmen (Tschechien) aufeinander treffen, es handelt sich um einen Punkt in der Nähe der Orte Regnitzlosau, Landkreis Hof, Bayern; Eichigt, Vogtlandkreis, Sachsen; und Hranice u Aše, Okres Cheb, heute Tschechische Republik.

Siehe auch [Bearbeiten]
Commons Commons: Dreiländereck – Bilder, Videos und Audiodateien

    * Dreikaisereck
    * Vierländereck
    * Dreiherrenstein
    * Tanzen im Dreiländereck
    * Dreigräben

Weblinks [Bearbeiten]

    * Liste aller Dreiländer als KML-Datei mit Koordinaten

Von „http://de.wikipedia.org/wiki/Dreil%C3%A4ndereck“
Kategorien: Politische Geographie | Grenze
