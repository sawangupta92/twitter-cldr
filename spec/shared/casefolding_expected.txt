dreiländereck
aus wikipedia, der freien enzyklopädie
wechseln zu: navigation, suche
dreiländereck d/f/ch am rheinhafen von basel
vaalserberg: dreiländereck (d/nl/b)
sevening: europadenkmal (d/b/l)
denkmal für das „schengener abkommen“ in schengen, luxemburg (d/l/f)
zittau: dreiländereck (d/cz/pl)
treriksröset: dreiländereck fin/n/s
berg ofen: grenzstein dreiländereck i/slo/a
monte roraima: dreiländereck brasilien, guyana, venezuela
markstein auf der argentinischen seite des dreiländerecks ra/br/py

ein dreiländereck – auch länderdreieck genannt – ist der geografische punkt, wo drei gleichrangige grenzen und auch drei gleich geartete territorien (länder, staaten, bundesländer) aufeinander treffen. bekannt werden solche dreiländerecke meist, wenn sie schon längere zeit bestehen oder aber, wenn drei kulturen aufeinander treffen.

meist ist diese stelle durch die jeweiligen staatsflaggen oder monumente markiert.

von den zurzeit 195 staaten auf der erde haben 134 einen solchen grenzpunkt. insgesamt existieren 157 dreiländerecke auf der erde. die häufigkeit des vorkommens von dreiländerecken ist abhängig von der anzahl der nachbarn, aber auch von der anzahl der seegrenzen. so haben inselnationen wie japan keines, wie auch staaten wie portugal oder dänemark, die nur ein nachbarland haben bzw. vollständig von einem staat umgeben sind (wie z. b. san marino). auch sind zwei nachbarländer kein hinreichendes kriterium, wie es z. b. bzgl. der usa in nordamerika ist. auch die fläche eines staats hat nur bedingt mit der anzahl an dreiländerecken zu tun: der staat mit den meisten ist china mit 15, während russland, der flächenmässig grösste staat der welt, nur acht hat.

österreich hat an seinen grenzen neun dreiländerecke (und ist damit der staat mit den meisten dreiländerecken innerhalb europas), deutschland sieben, die schweiz sechs und liechtenstein weist zwei dreiländerecke auf.
inhaltsverzeichnis
[verbergen]

    * 1 dreiländerecke weltweit
          o 1.1 europa
          o 1.2 mittel- und südamerika
          o 1.3 nordamerika
          o 1.4 afrika
          o 1.5 asien
    * 2 ehemalige dreiländerecke
          o 2.1 europa
    * 3 siehe auch
    * 4 weblinks

dreiländerecke weltweit [bearbeiten]

alphabetische liste von allen 46 internationalen dreiländerecken in europa:

europa [bearbeiten]

    * albanien – griechenland – mazedonien, am prespa-see
    * albanien – mazedonien – kosovo, südlich von prizren in kosovo
    * albanien – montenegro – kosovo, nördlich von bajram curri in albanien
    * andorra – frankreich – spanien (osteck), in den pyrenäen
    * andorra – frankreich – spanien (westeck), in den pyrenäen
    * belgien – deutschland – luxemburg, am europadenkmal an der our bei sevenig
    * belgien – deutschland – niederlande, auf dem vaalserberg westlich von aachen (war zeitweilig auch wegen neutral-moresnet ein vierländereck)
    * belgien – frankreich – luxemburg, bei longwy, pétange, aubange
    * bosnien-herzegowina – kroatien – montenegro, nordöstlich von gruda in kroatien
    * bosnien-herzegowina – kroatien – serbien, westlich von omjelovo in bosnien-herzegowina
    * bosnien-herzegowina – montenegro – serbien, westlich von priboj in serbien
    * bulgarien – griechenland – mazedonien, westlich von petritsch in bulgarien
    * bulgarien – griechenland – türkei, bei der griechischen gemeinde trigono, westlich des türkischen edirne
    * bulgarien – mazedonien – serbien, westlich von kjustendil in bulgarien
    * bulgarien – rumänien – serbien, östlich von negotin in serbien
    * deutschland – frankreich – luxemburg, bei perl, apach, schengen in der mosel
    * deutschland – frankreich – schweiz, bei basel, hüningen und weil am rhein im rhein
    * deutschland – polen – tschechien, südöstlich von zittau
    * deutschland – schweiz – österreich, im bodensee
    * deutschland – tschechien – österreich, am plöckenstein (1320 m), böhmerwald
    * estland – lettland – russland, westlich von stuborova in lettland
    * finnland – norwegen – russland, nordöstlich von ivalo in finnland
    * finnland – norwegen – schweden, schwedisch treriksröset, norwegisch treriksrøysa, finnisch kolmen valtakunnan rajapyykki, westlich von kilpisjärvi in finnland
    * frankreich – italien – schweiz, auf der bergspitze des mont dolent (3820 m) nahe dem mont blanc
    * italien – schweiz – österreich, nähe reschenpass an der „dreiländerecke“ (2.179 m)
    * italien – slowenien – österreich, bei arnoldstein auf dem gipfel des ofen (slow. peč) alias dreiländereck
    * kroatien – serbien – ungarn, nordöstlich von gruda in kroatien
    * kroatien – slowenien – ungarn, südöstlich von letenye in ungarn
    * lettland – litauen – weissrussland, nordöstlich von visaginas in litauen
    * lettland – russland – weissrussland, südöstlich von suskova in lettland
    * liechtenstein – schweiz – österreich (nordeck), im rhein bei sennwald
    * liechtenstein – schweiz – österreich (südeck), im rätikon am naafkopf (2571 m)
    * litauen – polen – russland (kaliningrad), westlich von wizajny in polen
    * litauen – polen – weissrussland, westlich von druskininkai in litauen
    * moldawien – rumänien – ukraine (nordeck), nordöstlich von dărăbani in rumänien
    * moldawien – rumänien – ukraine (südeck), östlich von galaţi in rumänien
    * polen – slowakei – tschechien, bei hrčava in tschechien und jaworzynka in polen
    * polen – slowakei – ukraine, nordöstlich von nova sedlica in slowakei
    * polen – ukraine – weissrussland, südöstlich von wlodawa in polen
    * rumänien – serbien – ungarn, südlich von kübekhaza in ungarn
    * rumänien – ukraine – ungarn, nordöstlich von bercu in rumänien
    * russland – ukraine – weissrussland, südöstlich von homel in weissrussland
    * slowakei – ukraine – ungarn, bei zahony in ungarn
    * slowakei – ungarn – österreich, bei deutsch jahrndorf in österreich, rajka in ungarn
    * slowakei – tschechien – österreich, bei hohenau an der march in österreich
    * slowenien – österreich – ungarn, östlich von tauka in österreich

die nachfolgende auflistung zeigt eine kleine auswahl existierender dreiländerecke in anderen kontinenten:

mittel- und südamerika [bearbeiten]

in mittelamerika existieren zwei dreiländerecke und in südamerika 13:

    * argentinien – bolivien – chile
    * argentinien – bolivien – paraguay
    * argentinien – brasilien – paraguay: hito tres fronteras, bekannt durch die iguaçu-wasserfälle
    * argentinien – brasilien – uruguay
    * mittelamerika: belize – guatemala – mexiko
    * bolivien – brasilien – paraguay
    * bolivien – brasilien – peru
    * bolivien – chile – peru
    * brasilien – kolumbien – peru: tres fronteras, im amazonas-regenwald mit einer dreistadt tabatinga, leticia, santa rosa mit mehr als 100.000 einwohnern
    * brasilien – kolumbien – venezuela
    * brasilien – französisch-guayana – surinam
    * brasilien – guyana – surinam
    * brasilien – guyana – venezuela: auf dem berg monte roraima
    * kolumbien – ecuador – peru
    * mittelamerika: guatemala – honduras – el salvador

nordamerika [bearbeiten]

da mit kanada und den vereinigten staaten nur zwei staaten in nordamerika neben einigen französischen inseln an der kanadischen ostküste existieren, kann es dort keine dreiländerecke geben.

die bundesstaaten der vereinigten staaten bilden 38 tri-state areas (dreistaatengebiete).

afrika [bearbeiten]

in afrika existieren 55 dreiländerecke:

    * südafrika – botswana – namibia
    * südafrika – botswana – simbabwe
    * südafrika – mosambik – simbabwe (siehe crook’s corner)
    * südafrika – mosambik – swasiland (nord)
    * südafrika – mosambik – swasiland (süd)
    * algerien – libyen – niger
    * algerien – libyen – tunesien
    * algerien – mali – mauretanien
    * algerien – mali – niger
    * algerien – marokko – west-sahara
    * algerien – mauretanien – west-sahara
    * angola – namibia – sambia
    * angola – republik kongo – demokratische republik kongo: cabinda
    * angola – demokratische republik kongo – sambia
    * benin – burkina faso – niger
    * benin – burkina faso – togo
    * benin – niger – nigeria
    * botswana – sambia – simbabwe
    * burkina faso – elfenbeinküste – ghana
    * burkina faso – elfenbeinküste – mali
    * burkina faso – ghana – togo
    * burkina faso – mali – niger
    * burundi – demokratische republik kongo – ruanda
    * burundi – ruanda – tansania
    * kamerun – gabun – äquatorial-guinea
    * kamerun – gabun – republik kongo
    * kamerun – nigeria – tschad
    * kamerun – zentralafrikanische republik – republik kongo
    * kamerun – zentralafrikanische republik – tschad
    * elfenbeinküste – guinea – liberia
    * elfenbeinküste – guinea – mali
    * dschibuti – eritrea – äthiopien
    * dschibuti – äthiopien – somalia
    * ägypten – libyen – sudan
    * eritrea – äthiopien – sudan
    * äthiopien – kenia – somalia
    * äthiopien – kenia – sudan
    * guinea – guinea-bissau – senegal
    * guinea – liberia – sierra leone
    * kenia – uganda – sudan
    * kenia – uganda – tansania
    * libyen – niger – tschad
    * libyen – sudan – tschad
    * malawi – mosambik – tansania
    * malawi – mosambik – sambia
    * mosambik – tansania – sambia
    * mosambik – sambia – simbabwe: bei zumbo
    * mali – mauretanien – senegal
    * niger – nigeria – tschad
    * uganda – demokratische republik kongo – ruanda
    * uganda – demokratische republik kongo – sudan
    * uganda – ruanda – tansania
    * zentralafrikanische republik – republik kongo – demokratische republik kongo
    * zentralafrikanische republik – demokratische republik kongo – sudan
    * demokratische republik kongo – tansania – sambia

asien [bearbeiten]

in asien existieren 42 dreiländerecke:

    * afghanistan – iran – pakistan
    * afghanistan – iran – turkmenistan
    * afghanistan – vr china – pakistan
    * afghanistan – vr china – tadschikistan
    * afghanistan – usbekistan – tadschikistan
    * afghanistan – usbekistan – turkmenistan
    * saudi-arabien – vereinigte arabische emirate – oman: umm az zamul
    * saudi-arabien – irak – jordanien
    * saudi-arabien – irak – kuwait
    * saudi-arabien – oman – jemen
    * armenien – aserbaidschan – georgien

mekong und goldenes dreieck

    * armenien – aserbaidschan – iran
    * armenien – aserbaidschan – iran
    * armenien – aserbaidschan – türkei
    * gaza-streifen – ägypten – israel
    * bangladesch – burma – indien
    * bhutan – vr china – indien (osten)
    * bhutan – vr china – indien (westen)
    * burma – vr china – indien
    * burma – vr china – laos
    * burma – laos – thailand: goldenes dreieck
    * kambodscha – laos – thailand
    * kambodscha – laos – vietnam
    * vr china – indien – nepal (osten)
    * vr china – indien – nepal (westen)
    * vr china – indien – pakistan
    * vr china – kasachstan – kirgisien
    * vr china – kasachstan – russland
    * vr china – laos – vietnam
    * vr china – mongolei – russland (osten)
    * vr china – mongolei – russland (westen)
    * vr china – nord-korea – russland in der region primorje
    * palästina – israel – jordanien (nord)
    * palästina – israel – jordanien (süden)
    * irak – iran – türkei
    * irak – jordanien – syrien
    * irak – syrien – türkei
    * israel – jordanien – syrien
    * israel – libanon – syrien: mount hermon
    * kasachstan – kirgisien – usbekistan
    * kasachstan – usbekistan – turkmenistan
    * kirgisien – usbekistan – tadschikistan

die dreiländerecke vr china – kasachstan – russland sowie vr china – mongolei – russland liegen im altai nur 50 kilometer voneinander entfernt.

ehemalige dreiländerecke [bearbeiten]

infolge von grenzverschiebungen kommt es vor, dass dreiländerecke zu normalen staatsgrenzen werden, aber dennoch ein markanter punkt bleiben:

europa [bearbeiten]

    * italien – schweiz – österreich; bis zur abtretung südtirols von österreich an italien, nähe stilfserjoch auf der dreisprachenspitze. es ist heute ein „dreispracheneck“ geblieben mit italienisch (lombardei), deutsch (südtirol) und rätoromanisch (schweiz).
    * br deutschland – ddr – tschechoslowakei; bis zur wiedervereinigung an der stelle, wo bayern, sachsen und böhmen (tschechien) aufeinander treffen, es handelt sich um einen punkt in der nähe der orte regnitzlosau, landkreis hof, bayern; eichigt, vogtlandkreis, sachsen; und hranice u aše, okres cheb, heute tschechische republik.

siehe auch [bearbeiten]
commons commons: dreiländereck – bilder, videos und audiodateien

    * dreikaisereck
    * vierländereck
    * dreiherrenstein
    * tanzen im dreiländereck
    * dreigräben

weblinks [bearbeiten]

    * liste aller dreiländer als kml-datei mit koordinaten

von „http://de.wikipedia.org/wiki/dreil%c3%a4ndereck“
kategorien: politische geographie | grenze
